src/libAtomVM/term.h

```diff
@@ -1503,6 +1503,16 @@ int term_fprint(FILE *fd, term t, const GlobalContext *global);
  */
 int term_snprint(char *buf, size_t size, term t, const GlobalContext *global);
 
+/**
+ * @brief Writes function name (without module and arity for external functions)
+ * as string to a buffer. Caller owns the buffer and must free it when done.
+ * @param fun the function term that will be converted to a string.
+ * @param global the \c GlobalContext.
+ * @param buf pointer to the buffer where the function name will be written.
+ * @returns the number of written characters.
+ */
+int term_local_function_name(term fun, const GlobalContext *global, char **buf);
```

Comment:
I'm not finding where this function is defined.

Reply:
Remnants from previous implementation, removed.